EdFi.Buzz.Installer/eng/README.md

```diff
@@ -0,0 +1,105 @@
+# Project Buzz Installer
+
+These scripts were made possible thanks to the Ed-Fi Alliance and the Michael and Susan Dell Foundation.
+
+## Description
+
+The instalation script downloads all the assets needed for the Ed-Fi Buzz. Artifacts are placed in the dist/ directory at the root of this project 
+
+It validates existence of NuGet Package Provider, Node JS, IIS, SQL Server and Postgres. Retrieves the latest release of several Ed-Fi Buzz NuGet packages. Downloads prequisites from the web. 
+
+The ETL requires a default, local instance of SQL Server, and the user executing the script must have sufficient permissions. Any existing Buzz databases on the build server should be backed-up and removed before running this build script. 
+
+## Installation
+- **Database:** This application provides the scripts to create the database structures necessary to execute Ed-Fi Buzz.
+- **ETL:** This application provides two ETL modules to load Google Form Survey data to the Buzz database, and to port data from the ODS data sources to Postgres.
+- **API:** GraphQL-based API for supporting the Ed-Fi Buzz solution.
+- **UI:** Project Buzz Web Application..
+
+## Prerequisites
+We recommend that the following prerequisites are installed on the machine that you are going to run the scripts on.
+
+1. Node Js LTS version https://nodejs.org/en/
```

Comment:
Does the installer load NodeJS or does it need to be in place beforehand?

Reply:
The installer requires it beforehand. FIF-300 is only checking that node is installed. Now, we do have the FIF-291 to use the Artifacts where we have to upload ALL of our NPM packages and these  nugets - we could make node part of that and control the version we use.